crates/bitwarden-exporters/src/cxf/editable_field.rs

```diff
@@ -0,0 +1,225 @@
+use bitwarden_vault::FieldType;
+use credential_exchange_format::{
+    EditableField, EditableFieldBoolean, EditableFieldConcealedString, EditableFieldString,
+    EditableFieldWifiNetworkSecurityType,
+};
+
+use crate::Field;
+
+/// Helper function to create a Field from any EditableField type
+pub(super) fn create_field<T>(name: String, field: &T) -> Field
+where
+    T: EditableFieldToField,
+{
+    Field {
+        name: Some(name),
```

Comment:
Almost every time we call this function, we're passing it a &str and calling .into() each time. It feels like it would be simpler to do something like:
```rust
pub(super) fn create_field<T>(name: impl Into<String>, field: &T) -> Field
where
    T: EditableFieldToField,
{
    Field {
        name: Some(name.into()),
```
And then functions could pass a &str or a String as needed.